discovery-provider/src/api/v1/tracks.py

```diff
@@ -19,3 +22,20 @@ def get(self, track_id):
             abort_not_found(encoded_id, ns)
         single_track = extend_track(tracks[0])
         return success_response(single_track)
+
+@ns.route("/<string:track_id>/stream")
+class TrackStream(Resource):
+    def get(self, track_id):
+        """Redirect to track mp3"""
+        encoded_id = decode_with_abort(track_id, ns)
+        args = {"track_id": encoded_id}
+        creator_nodes = get_track_user_creator_node(args)
+        if creator_nodes is None:
+            abort_not_found(encoded_id, ns)
+        creator_nodes = creator_nodes.split(',')
+        if not creator_nodes:
+            abort_not_found(encoded_id, ns)
+
+        primary_node = creator_nodes[0]
+        stream_url = urljoin(primary_node, 'stream/{}'.format(track_id))
+        return redirect(stream_url)
```

Comment:
does this do a 302 by default? (not a 301)

Reply:
Yes, it's a 302